collectors/python.d.plugin/adaptec_raid/metadata.yaml

```diff
@@ -112,12 +112,14 @@ modules:
                 enabled: false
               description: A basic example configuration per job
               config: |
+                ```
```

Comment:
The code block is not needed. They will be added during generation automatically.